subsys/bluetooth/services/ias/ias_client.c

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <stdint.h>
+#include <zephyr/sys/check.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/services/ias.h>
+
+#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_IAS_CLIENT)
+#define LOG_MODULE_NAME bt_ias_client
+#include "ias_internal.h"
```

Comment:
Hmm, technically there's nothing wrong with having the include on this line, but could you still please move the include of `#include "ias_internal.h"` outside the block that handles the debugging/logging? :)

Reply:
Sorry, as this is a draft I'm acting somewhat carelessly with some stuff, I'll move it of course.